db/schema.rb

```diff
@@ -855,6 +858,26 @@
     t.boolean  "active",                           default: true
   end
 
+  create_table "work_order_users", force: :cascade do |t|
+    t.integer  "work_order_id"
```

Comment:
I don't really remember, but if columns are indexed are they required by default? It seems that this (and other) columns don't have safety checks (not Null, not empty, exists?). I don't see anything about that in here and models don't have validations.

Reply:
You're right, in Rails 5 belongs_to associations are required and will raise validation errors if not there, but not Rails 4. I'll add a `required: true` to the associations (in my opinion this is a good habit to get into in a Rails 4 app due to the change)